shm/Store.cpp

```diff
@@ -0,0 +1,23 @@
+#include "Store.hpp"
+
+Response Store::buy(Cargo* cargo, size_t amount, Player* player){
```

Comment:
Code format issue - Check this file as you have `){` multiple times and it would be better with `) {`

Reply:
ok, fixed